crates/turborepo-lib/src/process/child.rs

```diff
@@ -878,8 +902,10 @@ mod test {
         };
 
         let output_str = String::from_utf8(output).expect("Failed to parse stdout");
+        let trimmed_out = output_str.trim();
+        let trimmed_out = trimmed_out.strip_prefix('\u{4}').unwrap_or(trimmed_out);
 
-        assert!(output_str.contains(input), "got: {}", output_str);
+        assert!(trimmed_out.contains(input), "got: {}", trimmed_out);
```

Comment:
Note: this is still a pretty loose assertion. We seem to be [hitting this](https://github.com/wez/wezterm/blob/main/pty/examples/narrow.rs#L53) causing the captured output to be "hello world\n^Dhello world\n" . I will investigate further and fix in a future PR.